downstream/assemblies/platform/assembly-platform-install-scenario.adoc

```diff
@@ -22,7 +22,7 @@ There are several supported installation scenarios for {PlatformName}. To instal
 * xref:ref-gateway-controller-hub-eda-ext-db[Single platform gateway, {ControllerName}, {HubName}, and {EDAcontroller} node with an external (installer managed) database]
 
 .Additional resources
-For a comprehensive list of pre-defined variables used in Ansible installation inventory files, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/red_hat_ansible_automation_platform_installation_guide/index#ref-ansible-inventory-variables[A.6. Ansible variables].
+For a comprehensive list of pre-defined variables used in Ansible installation inventory files, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/red_hat_ansible_automation_platform_installation_guide/index#ref-ansible-inventory-variables[Ansible variables].
```

Comment:
If the link is within this doc--should it be an x-ref?
If it's not within the doc--should the link include the {PlatformVers} attribute instead of 2.5?